renku/ui/cli/session.py

```diff
@@ -279,8 +333,26 @@ def stop(session_name, stop_all, provider):
     default=None,
     help="Session provider to use.",
 )
-def open(session_name, provider):
+@session_open_provider_options()
+def open(session_name, provider, **kwargs):
     """Open an interactive session."""
     from renku.command.session import session_open_command
 
-    session_open_command().build().execute(session_name=session_name, provider=provider)
+    session_open_command().build().execute(session_name=session_name, provider=provider, **kwargs)
+
+
+@session.command("ssh-setup")
+@click.option(
+    "existing_key",
+    "-k",
+    "--existing-key",
+    type=click.Path(exists=True, dir_okay=False),
+    metavar="<private key file>",
+    help="Existing private key to use.",
+)
+@click.option("--force", is_flag=True, help="Overwrite existing keys/config.")
```

Comment:
One more thing, I think `--yes` makes more sense in this context since we show a confirmation prompt to user.

Reply:
I had `--yes` and it was voted against in favor of `--force` by a 3:1 majority ... :man_shrugging: